src/pages/sunhoLee/Main/Fedd/Comment/Comment.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import CommentList from './CommentList';
+
+let id = 4;
+
+class Comment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      commentView: [],
+      comments: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      commentView: this.props.comments,
+    });
+  }
+
+  getCommentValue = e => {
+    this.setState({
+      comments: e.target.value,
+    });
+  };
+
+  uploadComment = e => {
+    if (this.state.comments.length > 0) {
+      this.state.commentView.push({
+        id: id,
+        userId: 'dltjsgho',
+        content: this.state.comments,
+      });
+
+      id++;
+    }
+    this.setState({ comments: '' });
+  };
+
+  handleEnter = e => {
+    e.preventDefault();
+    const { comments } = this.state;
+    if (comments.length > 0) {
+      this.uploadComment();
+    }
+  };
+
+  render() {
+    console.log(this.props.comments);
+    return (
+      <>
+        <div className="comment-view">
+          <ul className="comments">
+            {this.state.commentView.map(el => {
```

Comment:
el 이라는 매개변수명은 너무 광범위합니다. 항상 이름을 지을 때는 의미가 명확히 드러나는 이름으로 지어주세요
commentView라는 변수명도 조금 알아보기 힘드네요

state의 이름을 commentValue, comments 로 변경하는것이 좋아보입니다. 마찬가지로 여기서 매개변수명도 명확하게 변경하는 것이 좋아보입니다.
```suggestion
            {this.state.comments.map(comment => {
```